hacks/manifests/oathkeeper-external-accessrules.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  # The name comes from oathkeeper's subchart.
+  name: '{{ include "oathkeeper.fullname" . }}-rules'
```

Comment:
this is a static file, not a template, functions are not supported 😉

Reply:
No worries :) My idea for the CI refactor would be rather straightforward.
Current structure
```
hacks/
  values/
    oathkeeper.yaml
    keto.yaml
    ...
```

New structure
```
hacks/
  values/
    oathkeeper/
      case1.yaml
      case2.yaml
    keto/
      foo.yaml
```

This would allow us to hold multiple config options in the CI and tests all of them automatically :) 

In the quote you send i meant to deploy the cm as a static object, with a predefined `foo` name, and then feed that name as a variable in the values